soap/SoapHelperFunctions.php

```diff
@@ -1164,7 +1164,7 @@ function decrypt_string($string)
             $key = substr(md5($key), 0, 24);
             $iv = "password";
 
-            return openssl_decrypt($buffer, OPENSSL_CIPHER_3DES, $key, OPENSSL_ZERO_PADDING, $iv);
+            return openssl_decrypt(pack("H*", $buffer), 'des-ede3-cbc', $key, OPENSSL_NO_PADDING, $iv);
```

Comment:
It looks like some code duplication here? this function does exactly same as SoapHelperWebService::decrypt_string();
Maybe we can removing some code duplication instead fixing similar bugs in two identicaly same function.